security/login-error.php

```diff
@@ -23,7 +21,11 @@ function use_ambiguous_login_error( $error ): string {
 
 	// For lostpassword action, use different message.
 	if ( isset( $_GET['action'] ) && 'lostpassword' === $_GET['action'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
-		return FORGET_PWD_MESSAGE;
+		return esc_html__(
+			'If there is an account associated with the username/email address, you will receive an email with a 
+			link to reset your password.',
```

Comment:
Is there a reason not to use a single-line string?

Reply:
oops, fixed